### PR TITLE
Changed INSTALL.md

### PR DESCRIPTION
**Pull request by normanjaeckel**

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -59,13 +59,12 @@ but for most cases this is not recommended.
 Then run:
 
     $ docker-compose pull
-    $ docker-compose up
+    $ docker-compose up --detach
```
Comment (jsangmeister) [inline INSTALL.md:62]:
```suggestion
    $ docker-compose up -d
```
I know you like the verbose versions of commands more, but for the install documentation I would suggest the shorthand for an easier usage.

**Review by jsangmeister — CHANGES_REQUESTED** [review]

**Review by jsangmeister — APPROVED** [review]

File: INSTALL.md [hunk 2]
```diff
@@ -90,18 +94,19 @@ have several options to achieve this:
...
 
     $ docker-compose stop
 
 To remove all containers and networks run:
 
     $ docker-compose down
 
-To remove the database you have to remove the content of the `db-data`
-directory.
+To remove also the database (and loose all your data) run:
```
Comment (jsangmeister) [inline INSTALL.md:107]:
```suggestion
To remove also the database (and lose all your data) run:
```

File: INSTALL.md [hunk 3]
```diff
@@ -205,3 +210,24 @@ You have to customize all of these environment variables according to your mail
...
+
+
+
+## Database backup
+
+To get a dump of your (PostgreSQL) database run:
+
+    $ docker-compose exec --user postgres postgres pg_dump -U openslides --clean > dump.sql
+
+To restore your dump first shut down your instance if it is running:
```
Comment (jsangmeister) [inline INSTALL.md:222]:
```suggestion
To restore your dump, first shut down your instance if it is running:
```

File: INSTALL.md [hunk 4]
```diff
@@ -205,3 +210,24 @@ You have to customize all of these environment variables according to your mail
...
+
+    $ docker-compose down
+
+Then start only the postgres container and restore your dump:
+
+    $ docker-compose up --detach postgres
+    $ docker-compose exec --no-tty --user postgres postgres psql -U openslides < dump.sql
```
Comment (jsangmeister) [inline INSTALL.md:229]: You need to enter the password this way, or not? If so, I would add a hint for this

**Review by jsangmeister — CHANGES_REQUESTED** [review]

File: INSTALL.md (hunk 4 above)
Comment (normanjaeckel) [reply INSTALL.md:230]: There was no password request in my test.

**Comment by normanjaeckel** [comment]
Ready to merge.

**Review by jsangmeister — APPROVED** [review]

**Pull request merged**